robot_simulation:
  ros__parameters:
    dt: 0.001
    Ta: 0.070
    initial_state: [0.0, 0.0, 0.0, 0.0, 0.0]
